pymnt/cfg/tz1iZ9LkpAhN8X1L6RpBtfy3wxpEWzFrXz8j.yaml

```diff
@@ -0,0 +1,17 @@
+baking_address: tz1iZ9LkpAhN8X1L6RpBtfy3wxpEWzFrXz8j
```

Comment:
add this as discussed to the config pr

Reply:
@rvermootenct leave it here?

Reply:
oh yes, thank you. will add this in a new PR just so its not messy.